app/controllers/recommendations_controller.rb

```diff
@@ -1,27 +1,21 @@
 class RecommendationsController < ApplicationController
-  def New
-    @property = Property.new
+  def new
+    @recommendation = Recommendation.new
   end
 
   def create
-    @property = Property.new(property_params)
-    @property.save!
+    @recommendation = Recommendation.new(property_params)
```

Comment:
Does this have access to the postcode ID data from anywhere right now? Because postcode and postcode ID are not the same. User would enter their postcode, but this doesn't automatically link it to the correct postcode instance. First we would have to look up the correct postcode in the Postcode table via the postcode passed through the params.